contracts/contracts/sbtc-token.clar

```diff
@@ -110,6 +112,22 @@
 	)
 )
 
+(define-public (transfer-many (recipients (list 200 { amount: uint, sender: principal, to: principal, memo: (optional (buff 34)) })))
+	(fold complete-individual-transfer recipients (ok u0))
+)
+
+(define-private (complete-individual-transfer (individual-transfer { amount: uint, sender: principal, to: principal, memo: (optional (buff 34)) }) (helper-response (response uint uint)))
```

Comment:
Style Q: is using super-long lines the general style convention in clarity, or can this be multi-lined in an easy-to-read way?